site/en/docs/devtools/blog/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: "layouts/doc-post.njk"
+title: "DevTools Engineering Blog"
+description: "The Chrome DevTools engineering blog - by the engineers who build the DevTools."
+---
+
+Below is a list of everything that's been covered in the *Chrome DevTools Engineering Blog* series.
+
+* [CSS-in-JS support in DevTools](/blog/devtools-engineering-10)
+* [Improving DevTools startup time](/blog/devtools-engineering-9)
+* [Migrating Puppeteer to TypeScript](/blog/devtools-engineering-8)
+* [Debugging WebAssembly with modern tools](/blog/devtools-engineering-7)
+* [DevTools architecture refresh: migrating to Web Components](/blog/devtools-engineering-6)
+* [Simulating color vision deficiencies in the Blink Renderer](/blog/devtools-engineering-5)
+* [Puppetaria: accessibility-first Puppeteer scripts](/blog/devtools-engineering-4)
+* [How we built the Chrome DevTools WebAuthn tab](/blog/devtools-engineering-3)
+* [How we built the Chrome DevTools Issues tab](/blog/devtools-engineering-2)
+* [DevTools architecture refresh: migrating to JavaScript modules](/blog/devtools-engineering-1)
```

Comment:
Over time, I imagine we‘ll move away from a hardcoded list and instead use some kind of tagging/categorization system?

Reply:
+1. Will address this later after the migration.

Reply:
I think this can be removed with the changes I made in https://github.com/GoogleChrome/developer.chrome.com/pull/587/commits/dae8a553c6c8f900812acb1a9a03a5c707ba56f8

I think there's the potential to confuse users if we call this the devtools "blog" so I called it the engineering blog. However, I still think that this may confuse folks because this blog doesn't contain the What's New in DevTools posts and is instead targeted at a very unique audience that cares about how devtools implements specific features.

In other words, I think the majority of the people going to /docs/devtools will be web developers, and when they see a blog my hunch is they'd expect it to be targeted at them and contain What's New in DevTools. But instead they'd have to go to developer.chrome.com/blog to find that content.

I'm wondering if it might be better to tag both the engineering posts and the What's New in DevTools posts as `devtools` and have the blog link point to /tags/devtools. That way you're not splitting the content and if folks aren't interested in something they can skip over it.

Reply:
Taking another look, I just noticed the /docs/updates page which lists every What's New in DevTools post. I think I see now what y'all are trying to do. In that case I'd suggest looking at the change in dae8a55 and give all of those posts a `devtools` tag and have that entry in toc.yml link to /tags/devtools or /tags/new-in-devtools (whichever name you decide on).

Reply:
I will submit another PR for `new-in-devtools` tag.

Reply:
@chybie When you do that PR are you thinking of removing the `/updates` page? If so, it might be good to do it as part of this PR so Google Search doesn't index `/updates`. Alternatively we could ship a redirect so /updates redirects to /tags/new-in-devtools.